Makefile.nt

```diff
@@ -0,0 +1,36 @@
+# Makefile for Windows NT using Watcom compiler.
+#
+# wmake -f Makefile.nt
+# - builds libxmp.dll and its import lib (libxmp.lib)
+#
+# wmake -f Makefile.nt target=static
+# - builds the static library xmp_static.lib
+#
+# To disable module depacker functionality:
+#	wmake -f Makefile.nt USE_DEPACKERS=0
+#
+# To disable ProWizard functionality:
+#	wmake -f Makefile.nt USE_PROWIZARD=0
+#
+!ifndef target
+target = dll
+!endif
+
+USE_PROWIZARD	= 1
+USE_DEPACKERS	= 1
+
+CC = wcc386
+SYSTEM = nt_dll
+
+CFLAGS = -zq -bt=nt -bm -fp5 -fpi87 -mf -oeatxh -w4 -ei -zp8
+# newer OpenWatcom versions enable W303 by default.
+CFLAGS += -wcd=303
+# -5s  :  Pentium stack calling conventions.
+# -5r  :  Pentium register calling conventions.
+CFLAGS += -5s
+CFLAGS += -I"src/win32" -I"$(%WATCOM)/h/nt"
+
+OS_OBJS = src\win32\ptpopen.obj
```

Comment:
Lose the `src\win32\` path here: The line following it already handles it. Otherwise cross-compile on linux fails.